test/test_cuda.py

```diff
@@ -1009,9 +1009,17 @@ def test_var_large_input(self):
 
     def test_var_stability(self):
         tensor = torch.FloatTensor([2281.5, 2281.25]).cuda()
+
+        # Stability for inner dim
         self.assertEqual(tensor.var(0)[0], 0.03125)
+
+        # General stability
         self.assertEqual(tensor.var(), 0.03125)
 
+        # Stability for outer dimensions
+        tensor = tensor.unsqueeze(1)
```

Comment:
Are you sure this triggers the outerdim code? A lot of the THC code squeezes/combines dims before it chooses the kernel.

Reply:
Oh, I guess it does trigger outer-dim code path.

Reply:
Although performance-wise it would be better to squeeze 1-dimensions before deciding which kernel to run. But does not matter until it really becomes a bottleneck.